maquetado/scss/styles.scss

```diff
@@ -117,27 +119,16 @@
   line-height: 24px;
 }
 
-.navbar {
-  background-color: $white;
-  box-shadow: 0 4px 10px 0 $black-05;
-  height: 70px;
-  padding-left: 200px;
-  position: fixed;
-  top: 0;
-  width: 100%;
-  &::before {
-    content:'';
-    border-top: 4px solid $cerulean;
-    height: 4px;
-    left: 0;
-    position: absolute;
-    width: 100%;
-  }
+.input-description {
```

Comment:
Same, I would change this name to `input-label` for example. Also, you should add some margin bottom to this, it has no separation with your input field.